docset/windows/storage/New-Partition.md

```diff
@@ -60,31 +60,33 @@ Note: This cmdlet does not support creating dynamic volumes.
 ## EXAMPLES
 
 ### Example 1: Create a new partition on disk 1
-```
-PS C:\> New-Partition -DiskNumber 1 -UseMaximumSize -AssignDriveLetter
+```powershell
+PS C:\> New-Partition -DiskNumber 1 -UseMaximumSize -DriveLetter T
 ```
 
-This example creates a new partition on disk 1, using the maximum available space, and automatically assigning a drive letter.
+This example creates a new partition on disk 1 using the maximum available space and assigns a drive letter T.
 
 ### Example 2: Get all RAW disks, initialize the disks, partition, and format them
+```powershell
+PS C:\> Get-Disk | Where-Object PartitionStyle -Eq "RAW" | Initialize-Disk -PassThru | New-Partition -AssignDriveLetter -UseMaximumSize | Format-Volume
 ```
-This line gets all disk objects and then pipes the objects to the next command.
-PS C:\>Get-Disk |
 
-This line selects only objects where the PartitionStyle property value equals "RAW", and then pipes the objects to the next command.
-PS C:\>Where-Object PartitionStyle -Eq "RAW" |
+This example uses five cmdlets and the pipeline to get all disks, filter them for only RAW, unpartitioned disks, initialize the disks, partition the disks, and then to format them.
+
+### Example 3: Create a new EFI partition on GTP disk 2
+```powershell
+PS C:\> New-Partition -DiskNumber 2 -Size 500MB -GptType "{c12a7328-f81f-11d2-ba4b-00a0c93ec93b}"
+```
 
-This line initializes all Disk objects in the pipeline and then pipes the objects to the next cmdlet.
-PS C:\>Initialize-Disk -PassThru |
+This example creates a new EFI partition on disk 2 with size 500 MB.
```

Comment:
```suggestion
This example creates a new EFI partition on disk 2 with a size of 500 MB.
```

Reply:
Apologies @velkovb I seem to have missed this one.

Reply:
Ah, thanks for spotting that. Seems like I did half of it :)